CHANGELOG.md

```diff
@@ -7,7 +7,7 @@
 
 ### Breaking changes
 
-* Lorem ipsum.
+* New replication instruction: instr_AddRowWithKey
```

Comment:
From a *users* perspective, the instr_AddRowWithKey makes sense. That is the intended use case. It is technically correct, that the underlying machinery implements something that could be called instr_AddRowWithValue, but the intended usage matches instr_AddRowWithKey better. I'm fine with naming it based on the intended use.